pages/manual-content/search.html

```diff
@@ -21,6 +24,8 @@ <h2>Search this website</h2>
 	<button type="submit"  class="btn btn-primary">Search</button>	  
 </form>
 
+<div id="answersNow"></div>
```

Comment:
I don't think we want to release the Q&A stuff yet, this will make it live

Reply:
true, not sure we are ready for this. Unless we are ready to collect and act on all the feedback... hahaha.

Reply:
Q&A just goes to the Stay home page

Reply:
@abquirarte

Reply:
so it it still brings the new QA only on the stay home page

Reply:
Check the live site and see if that is what you want.

Reply:
This is going live with the quick answers on search results. We can go for it now and respond to feedback but we planned to user test and review our quick answer database for a few more days. I don't know if you have any intentional changes on this page or not.